ush/templates/FV3LAM_wflow.xml

```diff
@@ -511,8 +547,10 @@ MODULES_RUN_TASK_FP script.
     <dependency>
       <and>
         <or>
-          <taskdep task="&GET_EXTRN_LBCS_TN;"/>
+          <taskdep task="&GET_EXTRN_LBCS_TN;{{ uscore_ensmem_name }}"/>
+          {%- if not do_ensemble %}
```

Comment:
{%- if not do_ensemble %}

This is not required as it depends either GET_EXTRN_LBCS_TN **or** GET_EXTRN_LBCS_LONG_TN

Reply:
Removed quite a few " {% if not do_ensemble %}".

Reply:
Thanks a lot for addressing these picky comments.  However, the only left "{%- if not do_ensemble %}" can also be removed because as long as GET_EXTRN_LBCS_TN is completed, this task is good to go. GET_EXTRN_LBCS_LONG_TN under the <or> ..</or> section does not affect any functionality here. If you are not sure whether this will work, I have a simple demonstration xml at /lfs4/BMC/wrfruc/gge/rtma/xml/test.xml. So it is totally fine to remove this extra "{%- if not do_ensemble %}".

Reply:
Removed as suggested. Thanks!